### ISSUE TITLE
UIA registering as a screenreader leads to multiple issues with other programs by making them switch to accessibility mode

### ISSUE THREAD

**Issue by tomkahn** [Descolada/UIA-v2#66]
Hi Descolada,

First, let me say that I am grateful for your work on this project! It has been useful to me in automating Outlook specifically. For my purposes it is a better option than using the COM object.

I discovered recently that because of UIA some other programs see my program as a screen reader and switch to accessibility mode, which leads to all kinds of confusing issues for my users (programs look different, icons missing, text only etc. etc.).

**Is there a way to use the UIA library _without_ registering my script/program as a screen reader?**

The version of UIA-v2 that I am currently using still had a global variable called `IUIAutomationActivateScreenReader` that was set to 1 by default but as far as I can tell it didn't make a difference what its value was as `this.__UIA.SetScreenReader(1)` was never called anyway. (I set a breakpoint in VS Code but it never stopped, even when I interacted with Outlook via UIA.) Still my program was recognized as a screen reader.

It would be great to get your input on this. I would also be happy to pay for your time!

Best,
Thomas

PS: After sharing this issue in an AutoHotkey Telegram group, one experienced user wrote: "I've never heard of that before" Then a few minutes later: "Now that we are on this topic something similar has happened to me with the ahk UIA library and vscode. I think NOW I realize what was up: i opened UIA viewer and then vscode started acting up." Another user wrote: "I've had this experience once before and thought I was hacked, so I shutted down my PC. Now I know what it probably was." So it seems like this could actually be an issue!

**Comment by tomkahn** [comment]
Just tried it and restarted my PC. `IUIAutomationActivateScreenReader` is set to 0 right in the library and `GetScreenReader()` also returns 0 now. Outlook is still accessible via UIA, though! ✔️ So I guess this means that Outlook is not one of the programs that require the script to be registered as a screen reader?

**Comment by Descolada** [comment]
Yes, that is correct. 
I've now corrected UIA to not change the screen reader state if it was already enabled, and added some notes about the screen reader mode in the Wiki. Closing this issue now :)
